bitcoin-magazine/2021-04-23-taproot-activation-update.md

```diff
@@ -0,0 +1,411 @@
+---
+title: Taproot Activation Update (2021-04-23)
+transcript_by: Michael Folkson
+categories: ['podcast']
+tags: ['taproot']
+---
+
+Name: Sjors Provoost and Aaron van Wirdum
+
+Topic: Taproot Activation Update: Speedy Trial and the LOT=true client
+
+Location: Bitcoin Magazine (online)
+
+Date: April 23rd 2021
+
+Video: https://www.youtube.com/watch?v=SHmEXPvN6t4
```

Comment:
nit: valid markdown would be to put this in brackets like `<https://www.youtube.com/watch?v=SHmEXPvN6t4>`